content/src/main/content/jcr_root/apps/acs-commons/config.author/org.apache.sling.servlets.resolver.SlingServletResolver.config

```diff
@@ -0,0 +1,5 @@
+# Configuration created by Apache Sling JCR Installer
```

Comment:
This isn't going to work here -- we can't override the SlingServletResolver configuration in ACS AEM Commons

Reply:
Committed by mistake. I have to delete this file.